detox/android/detox/src/main/java/com/wix/detox/reactnative/ReactNativeLoadingMonitor.kt

```diff
@@ -13,7 +15,8 @@ private const val LOG_TAG = "DetoxRNLoading"
 open class ReactNativeLoadingMonitor(
         private val instrumentation: Instrumentation,
         private val rnApplication: ReactApplication,
-        private val previousReactContext: ReactContext?) {
+        private val previousReactContext: ReactContext?,
+        private val config: DetoxConfig = DetoxConfig.CONFIG) {
```

Comment:
you are assigning lateinit var, did u check that all flows that have access to this config is after it is being initialized?

Reply:
Yes - it must be set as early as in `Detox.runTest()`, and if that's not the case - I want the _uninitialized_ error to be thrown.